roles/redpanda_broker/tasks/start-redpanda.yml

```diff
@@ -35,17 +35,12 @@
     verbosity: 3
   when: redpanda is defined
 
-- name: Debug
-  ansible.builtin.debug:
-    msg: "{{ redpanda | from_json }}"
-    verbosity: 3
-  when: redpanda is defined
-
```

Comment:
Why removing this? Debugging when `redpanda` goes wrong is really hard, so this helps at verbosity 3.

Reply:
We already have another one that is just the string, seems unnecessary to duplicate it

Reply:
I added it back 🤷‍♀️

Reply:
The reason, for posterity, is that if the JSON doesn't parse, then it becomes obvious at this point. So the first one dumps the string, the second one dumps the JSON. Possibly not needed I guess, but if it fails on this line then you know why